app/services/IdentityService.scala

```diff
@@ -14,6 +13,8 @@ import play.api.libs.ws.{WS, WSRequest, WSResponse}
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
+case class UserId(id:String)
```

Comment:
Looks like a space optimisation that the app will not benefit too much from, since these values are not piling up to an overwhelming amount, but rather created per request and GC'ed after the response is sent out.

And remember :) ![](http://izquotes.com/quotes-pictures/quote-we-should-forget-about-small-efficiencies-say-about-97-of-the-time-premature-optimization-is-the-c-a-r-hoare-299203.jpg)